### PR TITLE
Display video audio preview for mobile devices

### PR DESCRIPTION
**Pull request by RidipDe**
**Description of changes:**
Display video audio preview for mobile devices

**Testing**

1. Have you successfully run `npm run build:release` locally?
Yes

2. How did you test these changes?
Ran it on my device and was able to see the preview

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: demos/browser/app/meetingV2/styleV2.scss [hunk 1]
```diff
@@ -425,6 +425,22 @@ svg {
   }
 }
 
+.video-preview {
```
Comment (simmkyu) [inline demos/browser/app/meetingV2/styleV2.scss:428]: 1. You can put this mobile view into the existing mobile `@media` block at line 361.
    ```
    @media (max-width: 575px) { 
        .video-preview { ... }
    }
    ```

File: demos/browser/app/meetingV2/styleV2.scss [hunk 2]
```diff
@@ -425,6 +425,22 @@ svg {
...
+  width:auto;
+  height:82px;
+  margin-left:35%;
+  margin-top: 7px;
+}
+
+@media (min-width: 576px){
```
Comment (simmkyu) [inline demos/browser/app/meetingV2/styleV2.scss:435]: 2. Then, remove this non-mobile `@media` and put `.videw-preview` outside. As a result, we can rely on one `@media` query.

File: demos/browser/app/meetingV2/styleV2.scss [hunk 3]
```diff
@@ -425,6 +425,22 @@ svg {
   }
 }
 
+.video-preview {
+  width:auto;
```
Comment (simmkyu) [inline demos/browser/app/meetingV2/styleV2.scss:429]: Add a space after `:`. I understand that we don't run ESLint on demo files. Let's follow coding conventions from the existing code.

File: demos/browser/app/meetingV2/styleV2.scss [hunk 4]
```diff
@@ -425,6 +425,22 @@ svg {
...
+  height:82px;
+  margin-left:35%;
+  margin-top: 7px;
+}
+
+@media (min-width: 576px){
+  .video-preview {
+    width:100%;
+    height:82px;
+    margin-left:0%;
```
Comment (simmkyu) [inline demos/browser/app/meetingV2/styleV2.scss:439]: Recommend using the same unit for zero, or you can just use `0` without any unit.
```
margin-left: 0;
margin-top: 0;
```

File: demos/browser/app/meetingV2/styleV2.scss (hunk 3 above)
Comment (RidipDe) [reply demos/browser/app/meetingV2/styleV2.scss:429]: done

File: demos/browser/app/meetingV2/styleV2.scss (hunk 2 above)
Comment (RidipDe) [reply demos/browser/app/meetingV2/styleV2.scss]: yeah sounds good

**Pull request merged**